vulture_os/system/cluster/models.py

```diff
@@ -496,7 +493,10 @@ def api_request(action, config=None, node=None, internal=False):
         :param node:      The node to set the action to
         :param internal:  Is this request internal ? Means that it will not be shown to the admin
         :return:
-            { 'status': True, 'message': 'A meaningful message' }
+                for no node is specified:
+                { 'status': True, 'message': 'A meaningful message' }
+                for specific node:
+                { 'status': True, 'message': 'A meaningful message', instance: messagequeue_object_created }
```

Comment:
It would be better/more generic to always return a list of instances (when the call succeeds), and then handle those messages with a for loop (or an index access) externally, even when a single node is targeted